setup.py

```diff
@@ -0,0 +1,114 @@
+"""
+Neuraxle
+====================================
+This is the core of Neuraxle. Most pipeline steps derive (inherit) from those classes. They are worth noticing.
+..
+    Copyright 2019, Neuraxio Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+from setuptools import setup, find_packages
+
+from neuraxle import __version__ as _VERSION
+
+with open('README.md') as _f:
+    _README = _f.read()
+
+setup(
+    name='neuraxle_tensorflow',
+    version=_VERSION,
+    description='TensorFlow steps, savers, and utilities for Neuraxle. Neuraxle is a Machine Learning (ML) library for building neat pipelines, providing the right '
+                'abstractions to both ease research, development, and deployment of your ML applications.',
+    long_description=_README,
+    classifiers=[
+        "Development Status :: 4 - Beta",
```

Comment:
I'd put `alpha` here instead: 
- `Development Status :: 3 - Alpha`

See: https://pypi.org/classifiers/

Also, for all the classifiers below, I'd remove some. Neuraxle is truly general-purpose, whereas here there are less classifiers. For instance, I'd remove the following classifiers from the present new project: 
```
        "Topic :: Software Development :: Pre-processors",
        "Topic :: Software Development :: Quality Assurance",
        "Topic :: Software Development :: Testing",
        "Topic :: Utilities",
```